frame/bet/src/lib.rs

```diff
@@ -0,0 +1,984 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
```

Comment:
New file.

Reply:
dunno how it got dirty, I used master afaik :/